api/.gitignore

```diff
@@ -12,6 +12,7 @@
 /test/version_tmp/
 /tmp/
 /var
+/usr
```

Comment:
I don't think we want this.  We have example scripts in `usr/share/` and we may wish to add new scripts in the future.  This will likely get in the way of that in an annoying manner.

Reply:
7239615abcc35c9c759c0db4defa68c593cd6a6e